Samples/word-citation-management/README.md

```diff
@@ -0,0 +1,140 @@
+---
+page_type: sample
+urlFragment: word-citation-management
+products:
+- office-word
+- m365
+languages:
+- javascript
+extensions:
+  contentType: samples
+  technologies:
+  - Add-ins
+  createdDate: 11/15/2023 4:00:00 PM
+description: "Shows how to manage citations in a Word document."
+---
+
+# Manage citations in a Word document
+
+## Summary
+
+This sample shows the basics of managing citations with an add-in in a Word document. The add-in uses the [insertEndnote API](https://learn.microsoft.com/javascript/api/word/word.range?view=word-js-1.5#word-word-range-insertendnote-member(1)) to list inserted citations at the end of the document.
+
+For documentation related to this sample, see [Citation management article](https://learn.microsoft.com/office/dev/add-ins/word/citation-management).
+
+## Description
+
+The user updates their Word document to include their citations and quotations. The user selects their bibliography file (.bib) through the add-in UI. The add-in lists the references found in the .bib file. The user selects the text in the document that they want to cite then picks the option from the reference list displayed in the add-in UI. The inserted citations are listed at the end of the document.
+
+The sample also depends on [@orcid/bibtexParseJs](https://github.com/ORCID/bibtexParseJs#readme), a JavaScript library for parsing the .bib file.
+
+![Citation management add-in task pane.](./assets/word-citation-management.png)
+
+## Applies to
+
+- Word on Windows
+- Word on Mac
+- Word on the web
+
+## Prerequisites
+
+- Office connected to a Microsoft 365 subscription (including Office on the web).
+- [Node.js](https://nodejs.org/) version 16 or greater.
+- [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) version 8 or greater.
+
+- [@orcid/bibtexParseJs](https://github.com/ORCID/bibtexParseJs#readme).
+
+## Solution
+
+| Solution | Authors |
+|----------|-----------|
+| How to manage citations in a Word document | Microsoft |
+
+## Version history
+
+| Version  | Date | Comments |
+|----------|------|----------|
+| 1.0 | 11-15-2023 | Initial release |
+
+## Run the sample
+
+1. Clone or download this repo.
+
+1. Go to the **Samples\word-citation-management** folder via the command line.
+
+1. Run `npm install`.
```

Comment:
Maybe it's just my local machine, but I had a few deprecation warnings:

```
npm WARN deprecated readdir-scoped-modules@1.1.0: This functionality has been moved to @npmcli/fs
npm WARN deprecated debuglog@1.0.1: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
npm WARN deprecated @npmcli/move-file@1.1.2: This functionality has been moved to @npmcli/fs
npm WARN deprecated @npmcli/move-file@2.0.1: This functionality has been moved to @npmcli/fs
```

Reply:
I'll check into it

Reply:
per our offline discussion, I'll deal with this in a subsequent PR if need be